assets/css/main.css

```diff
@@ -277,6 +277,383 @@ code {
 	--number: #b5cea8;
 }
 
+body
```

Comment:
You just duplicated in the entire (old) stylesheet. Please fix this.

Reply:
Yeah, i just realized, i fucked up the git config and had to move the things to a new repository